app/controllers/admin/budget_investments_controller.rb

```diff
@@ -15,6 +15,7 @@ class Admin::BudgetInvestmentsController < Admin::BaseController
   def index
     respond_to do |format|
       format.html
+      format.js { render layout: false }
```

Comment:
Why is this necessary?

Reply:
So we can make use of the advanced filters without reloading the entire page and just update what's needed —see GIF above.